from domain.exceptions import DomainError


class MatchesDomainError(DomainError):
    ...
